src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -106,6 +107,9 @@ public void createColony(@NotNull final World w, final BlockPos pos, @NotNull fi
         colony.setName(colonyName);
         colony.getPermissions().setPlayerRank(player.getGameProfile().getId(), Rank.OWNER, w);
 
+        colony.getPackageManager().addGlobalSubscriber((EntityPlayerMP) player);
```

Comment:
Why are we adding the player to both here?

Reply:
because owners do receive global messages and upon creating the colony the player is within the colony, so he is added to the close subscribers aswell

Reply:
Okey